README.md

```diff
@@ -1,6 +1,6 @@
-#powershell
+# PowerShell
```

Comment:
Yep, that's correct @ferventcoder . Module names (powershell) are lowercased, the technology names are capitalized as per their usual style (PowerShell).